src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -218,6 +218,178 @@ void uct_rc_mlx5_iface_common_tag_cleanup(uct_rc_mlx5_iface_common_t *iface,
 #endif
 }
 
+
+#if HAVE_IBV_EXP_DM
+const uint32_t uct_rc_mlx5_dm_key = 1245; /* magic number */
+
+typedef struct {
+    uct_mlx5_dm_data_t *dm;
+    int                count;
+} uct_iface_dm_mp_priv_t;
+
+typedef struct ucx_vma_mlx5_dm {
+    struct ibv_exp_dm  ibv_dm;
+    size_t             length;
+    uint64_t           *start_va;
+} ucx_vma_mlx5_dm_t;
```

Comment:
the name does not look right: 
- we do not use `ucx`
- `vma`

Reply:
also, where do you use it?

Reply:
renamed, used in va_address initialization